Firestore/Source/API/FIRDocumentSnapshot.mm

```diff
@@ -18,6 +18,7 @@
 
 #include <utility>
 
+#import "FIRFirestore.h"
```

Comment:
This is in the wrong spot. Group with `FIRFirestoreSettings.h` below.

Reply:
Done.